A32NX/html_ui/Pages/VCockpit/Instruments/A320_Neo/Map/A32NX_MapInstrument.js

```diff
@@ -259,6 +259,9 @@ class MapInstrument extends ISvgMapRootElement {
                 this.showFlightPlan = true;
                 this.updateFlightPlanVisibility();
             }
+            if (newValue == "false") {
```

Comment:
can we make this a Boolean instead of a string? By the looks of it it seams very obvious, but haven't looked at the entire code.

Reply:
This is in line with the rest of the syntax on this page, so I am not inclined to change it.